src/number.cr

```diff
@@ -211,8 +211,13 @@ struct Number
   # ```
   def round(digits, base = 10)
     x = self.to_f
-    y = base ** digits
-    self.class.new((x * y).round / y)
+    if digits < 0
+      y = base ** (digits * -1)
```

Comment:
It should: https://github.com/crystal-lang/crystal/issues/4549 . I just fixed it in master. Let's use `(-digits)` for now :-)